.github/workflows/uploadFirrtlReleaseArtifacts.yml

```diff
@@ -1,11 +1,38 @@
-name: Upload Firrtl Binaries
+name: Upload Firrtl Release Artifacts
 
 on:
   release:
     types: [created]
   workflow_dispatch:
 
 jobs:
+  publish-sources: 
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-20.04
+    steps:
+      # Clone the CIRCT repo and its submodules. Do shallow clone to save clone
+      # time.
+      - name: Get CIRCT and LLVM
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+          submodules: "true"
```

Comment:
Does the shallow clone include the tag? Since this is only done on a tag it _should_ but want to make sure the sources correct grab the version.